src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -3479,4 +3480,42 @@ WindowID MessageHelper::ExtractWindowIdFromSmartObject(
   return mobile_apis::PredefinedWindows::DEFAULT_WINDOW;
 }
 
+void MessageHelper::AddDefaultParamsToTireStatus(
+    ApplicationSharedPtr app, smart_objects::SmartObject& response_from_hmi) {
+  const utils::SemanticVersion max_version_with_mandatory_params(7, 1, 0);
+
+  if (!app) {
+    SDL_LOG_ERROR("Application not found");
+    return;
+  }
+
+  if (app->msg_version() > max_version_with_mandatory_params) {
```

Comment:
If agreed to previous comment:

```suggestion
  if (app->msg_version() >= max_version_with_mandatory_params) {
```

Reply:
@JackLivio fixed in https://github.com/smartdevicelink/sdl_core/pull/3583/commits/4db937435cc5a31454d32cced0abb9d9996ecec1